tests/page-objects/default/analytics/analytics.wdio.page.js

```diff
@@ -4,6 +4,8 @@ const TARGET_UNMET_COLOR = '#000000';
 
 const goToTargets = () => browser.url('/#/analytics/targets');
 
+const goToTargetAggregates = () => browser.url('/#/analytics/target-aggregates');
```

Comment:
We already have a function to navigate [to aggregate targets, here. ](https://github.com/medic/cht-core/blob/4e73a79c71c5e37d892e975052d4bb2a8961e7e8/tests/page-objects/default/targets/target-aggregates.wdio.page.js#L26) 
Use that one because it handles loaders for you already. Then, revert the changes in this file.